ndcube/ndcube_sequence.py

```diff
@@ -239,6 +244,67 @@ def explode_along_axis(self, axis):
         # creating a new sequence with the result_cubes keeping the meta and common axis as axis
         return self._new_instance(result_cubes, common_axis=new_common_axis, meta=self.meta)
 
+    def __get_sequence_axes_wcs(self):
```

Comment:
Don't think the double underscore is really needed.

```suggestion
    def _get_sequence_axes_wcs(self):
```